src/seth/libexec/seth/seth-index

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+### seth-index -- Prints the slot number for the specified mapping type and input data
+### Usage: seth index <fromtype> <totype> <fromvalue> <slot> [<lang>]
+###
+### Prints the slot number for the specified mapping type and input data. For example,
+### the balances mapping in DAI is slot 2, so to find the slot where the balance 
+### is stored for <account>, use
+###     seth-index address uint256 <account> 2
+
+set -e
+
```

Comment:
Can we print usage info we don't have the expected number of params, right now if I run `seth index` it shows a fairly instrucable error from `hevm`:

```
> seth index
hevm: wrong number of arguments:0: []
CallStack (from HasCallStack):
  error, called at hevm-cli/hevm-cli.hs:984:11 in main:Main
```

Reply:
Done in ba95b045